code/modules/overmap/exoplanets/turfs.dm

```diff
@@ -265,6 +265,5 @@
 		A.forceMove(T)
 		if(isliving(A))
 			var/mob/living/L = A
-			if(L.pulling)
-				var/atom/movable/AM = L.pulling
-				AM.forceMove(T)
+			for(var/obj/item/grab/G in L.get_active_grabs())
+				G.affecting.forceMove(T)
```

Comment:
It won't be dropped on the moved event from the `A.forceMove` above?

Reply:
Nope, works for stairs too.